ansible/roles/ocp-workload-iot-demo/tasks/workload.yml

```diff
@@ -46,117 +47,184 @@
         state: started
         delay: 0
         connect_timeout: 2
         timeout: 3
-      loop: "{{ range(mqtts_port|int + 1, 32102, 1)|list }}"
+      loop: "{{  range(mqtts_port|int + 1, 32102, 1)|list  }}"
       when: portcheck.failed == false
       register: portcheck
-  rescue:    
+  rescue:
     - set_fact:
-        mqtts_port: "{{ portcheck.results|selectattr('failed', 'defined')|selectattr('failed')|map(attribute='item')|first}}"
-    - debug: msg="MQTTS Port Assignment is {{ mqtts_port }}"
+        mqtts_port: >-
+          {{  portcheck.results|selectattr('failed', 'defined')|selectattr('failed')|map(attribute='item')|first }}
+    - debug: msg="MQTTS Port Assignment is {{  mqtts_port  }}"
     # Use to force fail on rescue since we short circuit the failure by handling in rescue
     #- command: /bin/false
 
 - name: Create project for IoT Demo
   shell: |
-         oc new-project {{ocp_project}} \
-         --display-name="IoT Demo - {{guid}}" \
+         oc new-project {{ ocp_project }} \
+         --display-name="IoT Demo - {{ guid }}" \
          --description="Industry 4.0 IoT Demo"
   ignore_errors: true
 
 - name: "Label namespace"
-  command: "oc label namespace {{ocp_project}} AAD='{{guid}}'"
+  command: "oc label namespace {{ ocp_project }} AAD='{{ guid }}'"
 
 - name: Add anyuser
-  shell: "oc adm policy add-scc-to-user anyuid -z default -n {{ocp_project}}"
+  shell: "oc adm policy add-scc-to-user anyuid -z default -n {{ ocp_project }}"
 
 - name: Add Quay Image Pull Secret for Eurotech Images
-  shell: "oc create -f /tmp/{{guid}}/redhat-iot-rhpds-secret.yml -n {{ocp_project}}"
+  shell: "oc create -f /tmp/{{ guid }}/redhat-iot-rhpds-secret.yml -n {{ ocp_project }}"
 
 - name: Add Quay Image Pull Secret for Eurotech Images
-  shell: "oc secrets link default redhat-iot-rhpds-pull-secret --for=pull -n {{ocp_project}}"
+  shell: "oc secrets link default redhat-iot-rhpds-pull-secret --for=pull -n {{ ocp_project }}"
 
 - name: Add MariaDB Secret
-  shell: "oc create secret generic ec-db --from-literal=name=ecdb --from-literal=username=redhat --from-literal=password=RedHat123 -n {{ocp_project}}"
+  shell: >-
+    oc create secret generic ec-db
+    --from-literal=name=ecdb
+    --from-literal=username=redhat
+    --from-literal=password=RedHat123
+    -n {{ ocp_project }}
 
 - name: Ensure dir exists
   file:
-    path: /tmp/{{guid}}/certs/
+    path: /tmp/{{ guid }}/certs/
     state: directory
 
 - name: Generate Self Signed Certs
-  command: >
+  command: >-
     openssl req -new
       -x509
       -nodes
-      -days {{ item.days | default(3650) }}
-      -subj "/C={{ item.country | default('') }}/ST={{ item.state | default('') }}/L={{ item.city | default('') }}/O={{ item.organization | default('') }}/OU={{ item.unit | default('') }}/CN={{ item.name }}/emailAddress={{ item.email | default('') }}"
+      -days {{  item.days | default(3650)  }}
+      -subj "/C={{  item.country | default('')
```

Comment:
fix for readability. close `}}` on the same line it is open, clean up spaces, etc.

Reply:
Will not fit on one line (closing braces must go on next line for spacing, otherwise entire string will need to go on one line)